action.yml

```diff
@@ -6,78 +6,173 @@ inputs:
+    description: Specifies whether mock db objects should be used to fill out dependencies. If set to true mock-db-object-dependency-list must also be set, otherwise an error will occur. The expected value is true or false.
     required: false
+    default: 'false'
+  mock-db-object-dependency-list:
+    description: A json string containing a list of objects with the name of the dependency package, the version, and the url where the package is stored.
+    required: true
   incremental:
-    description: Specifies whether to drop and recreate the database before building, or apply to the current database. The expected value is true or false.
+    description: Specifies whether to drop and recreate the database before building, or apply to the current database. The expected value is true or false. If true, the create-database-file property will not be used.
     required: false
     default: 'false'
+  create-database-file:
+    description: The file path to the sql file that initializes the database. This script will only be run if the incremental property is false.
+    required: false
   run-tests:
-    description: Specifies whether or not to run tests.
+    description: Specifies whether or not to run tests. The expected values are true and false. If true, test-files-path should also be set.
     required: false
     default: 'false'
+  test-files-path:
+    description: The path to the files with tSQLt tests.
+    required: false
   drop-db-after-build:
-    description: Specifies whether or not to drop the database after building. Set this to false if other steps in the job rely on the database existing.
+    description: Specifies whether or not to drop the database after building. The expected values are true and false. Set this to false if other steps in the job rely on the database existing.
     required: false
-    default: 'true'
+    default: 'false'
   should-validate-migrations:
     description: Determines whether flyway will validate the migration scripts before running them.
     required: true
     default: 'false'
   seed-data:
-    description: A switch specifying whether or not to seed data into the database.
+    description: A switch specifying whether or not to seed data into the database. The expected values are true and false. If true, seed-data-files-path must also be set.
     required: false
     default: 'false'
-  db-username:
-    description: The username to log into the database with. If not set, then integrated security will be used.
+  test-timeout:
+    description: An optional setting for the allowed wait time, in seconds, for the tests to execute. If tests sometimes hang, or shouldn't take longer than a certain amount of time, this parameter can be helpful.
     required: false
-  db-password:
-    description: The password associated with the db-username used for login.
+    default: '300'
+  seed-data-files-path:
+    description: The path to the files with seeding database.
     required: false
```

Comment:
This is nitpicky but could you move this one up under the seed-data property?  All of the other flags have their file paths right under them.

Reply:
Not nitpicky at all! I was purposeful in the ordering and must have missed this one. Thanks for pointing it out!